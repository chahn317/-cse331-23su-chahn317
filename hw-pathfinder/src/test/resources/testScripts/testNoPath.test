# Test case where there is no path between two nodes

CreateGraph graph1

AddNode graph1 A
AddNode graph1 B
AddNode graph1 C
AddNode graph1 D

# no edges

FindPath graph1 A B

# add edges

AddEdge graph1 A B 1.0
AddEdge graph1 A C 1.0

FindPath graph1 A D

FindPath graph1 B D
