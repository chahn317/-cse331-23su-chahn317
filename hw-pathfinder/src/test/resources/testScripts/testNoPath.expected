# Test case where there is no path between two nodes

created graph graph1

added node A to graph1
added node B to graph1
added node C to graph1
added node D to graph1

# no edges

path from A to B:
no path found

# add edges

added edge 1.000 from A to B in graph1
added edge 1.000 from A to C in graph1

path from A to D:
no path found

path from B to D:
no path found
